src/Uno.UI/Helpers/MarkupHelper.cs

```diff
@@ -74,21 +74,22 @@ public static void SetVisualStateLazy(VisualState target, Action builder)
 		public static void SetVisualTransitionLazy(VisualTransition target, Action builder)
 			=> target.LazyBuilder = builder;
 
-		public static IXamlServiceProvider CreateParserContext(object? target, ProvideValueTargetProperty? property)
+		public static IXamlServiceProvider CreateParserContext(object? target, Type propertyDeclaringType, string propertyName, Type propertyType) =>
+			CreateParserContext(target, propertyDeclaringType, propertyName, propertyType);
```

Comment:
It's recursive otherwise

```suggestion
			CreateParserContext(target, propertyDeclaringType, propertyName, propertyType, null);
```